src/importlinter/contracts/forbidden.py

```diff
@@ -30,6 +30,10 @@ class ForbiddenContract(Contract):
         - unmatched_ignore_imports_alerting: Decides how to report when the expression in the
                              `ignore_imports` set is not found in the graph. Valid values are
                              "none", "warn", "error". Default value is "error".
+        - as_packages:       Whether to treat the source and forbidden modules as packages. If
```

Comment:
Could we also update the public-facing documentation - see docs/contract_types.rst?